types/amap-js-api/test/arryBounds.ts

```diff
@@ -0,0 +1,18 @@
+import {
```

Comment:
how does this pass CI? I thought it required a single `amap-js-api-test.ts` that imports all the other test files.

Reply:
i merged all test into "amap-js-api-tests.ts" now